<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>610</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="HLayoutKeywords">
      <item>
       <widget class="QLabel" name="LabelKeywords">
        <property name="text">
         <string>Search by keywords:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditKeywords">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Type keywords here to filter TB</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSearch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="StackCentral">
      <widget class="QWidget" name="PageLog">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="TextLog">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PageTable">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="TableTB">
          <column>
           <property name="text">
            <string>Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Category</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>RK number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tech. Publication</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Release date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Registered by</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Replaces</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Replaced by</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Keywords</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
